README.md

```diff
@@ -1,2 +1,114 @@
-# Covid-19-Data-Hub
-This is a webapp that display metric data from an API for Covid-19 cases globally. Interactions  Home page  When the page loads, the webapp shows the list of categories that could be filtered out by some parameter. Along with the category name, it displays some numeric values per category that come from the API. When the user clicks (or taps) on a category item, the application navigates to the details page.  Details page  In the details page, the webapp retrieves data from the API to show detailed data in the category. When the user clicks on the "Back" button (&lt;), the user navigates to the home page.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# COVID-19 DATA HUB
+
+> This is a webapp that display metric data from an API for Covid-19 cases globally
+
+![screenshot](./src/images/homepage.png)
+![screenshot](./src/images/Countrydetails.png)
+![screenshot](./src/images/mobile.png)
+
+Interactions
+
+Home page
+
+When the page loads, the webapp shows the list of categories that could be filtered out by some parameter.
+Along with the category name, it displays some numeric values per category that come from the API.
+When the user clicks (or taps) on a category item, the application navigates to the details page.
+
+Details page
+
+In the details page, the webapp retrieves data from the API to show detailed data in the category.
+When the user clicks on the "Back" button (<), the user navigates to the home page.
+
+## Built With
+
+- Javascript, HTML, CSS
+- React
+- Redux
+- API
+
+## Live Demo
+
+[Live Demo Link](https://cool-bavarois-7aa72a.netlify.app/)
+
+## Project Description
+
+Click here to watch a [video](https://www.loom.com/share/5b85672c9c3e4965b969d3bae797fdb1) that gives a brief description of the project
```

Comment:
- [ ] Please include in your video a part of your code you are proud of as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=you%20should%20also%20highlight%20some%20interesting%20piece%20of%20code%20or%20something%20you%20built%20that%20you%20are%20very%20proud%20of.)